WebClock/CountBalance.cs

```diff
@@ -27,5 +27,42 @@ public static BalanceDto CountWorkTime(List<ClockInOut> datesIn, List<ClockInOut
                 MinutesLeft = totalLeftTime.Minutes
             };
         }
+        public static BalanceDto CountWorkTimeCurrent(List<ClockInOut> datesIn, List<ClockInOut> datesOut)
+        {
+            var totalWorkTime = new TimeSpan();
+            var hoursToWorkInMonthToThisDay = new TimeSpan(0, Convert.ToInt32(GetBusinessDaysInThisMonth() * 8), 0, 0);
+            if (datesIn.Count > datesOut.Count)
+            {
+                datesOut.Add(new ClockInOut { UserId = 1, Date = DateTime.UtcNow, Type = ClockType.Out });
```

Comment:
Domyślam się, że tu chodzi o uzupełnienie godziny wyjścia w dniu "dzisiejszym" - żeby dało się zrobić pary wejść-wyjść.
Jednak coś mi się tutaj nie podoba - bo dobijając takie "fejkowe" dane zakłamujemy rzeczywistość.
Chyba lepiej byłoby obciąć 1 wejście (z "dzisiaj rana") i mieć pary do policzenia bilansu + informacja o godzinie dzisiejszego wejścia.

W dodatku takie rozwiązania nie zadziała, jeśli ilość wejść będzie większa od ilości wyjść o więcej niż 1 ;-)

Reply:
Dobra - widzę , że jest to zrobione z pełnym zamysłem - łącznie z testami.

Zostawmy to tak jak jest - dzisiaj działamy na happy-path - i takie rozwiązanie mogę zaakceptować.